src/main/java/org/altbeacon/beacon/service/RunningAverageRssiFilter.java

```diff
@@ -0,0 +1,78 @@
+package org.altbeacon.beacon.service;
+
+import org.altbeacon.beacon.logging.LogManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Iterator;
+
+/**
```

Comment:
Javadoc should say more about what this does like the javadoc for ArmaRssiFilter